src/background/actions/action-payloads.ts

```diff
@@ -128,6 +129,10 @@ export interface SetBugServicePayload extends BaseActionPayload {
     bugServiceName: string;
 }
 
+export interface SaveIssueFilingSettingsPayload extends SetBugServicePayload {
```

Comment:
Having read more of the code now, and particularly thinking about the `extends SetBugServicePayload` here, I'm thinking that my earlier suggestion about renaming went the wrong way. It looks like this message should have the same semantics as `SetBugService` but also set the bundle of properties. If that's correct, I would suggest the following:

- Message to be named `SetBugServiceAndProperties`
- Payload to be named `SetBugServiceAndPropertiesPayload`
- Property within payload to be named `bugServiceProperties` to match type `BugServiceProperties`
- Methods related to `SetBugServiceAndProperties` should immediately follow their `SetBugService` analogs.

Thoughts?

Reply:
Make sense!